### PR TITLE
Do not make __repr__ and __str__ output latex

### PR DESCRIPTION
**Pull request by eric-wieser**
These should be used for plaintext output.
For now, we use the `__ga_print_str__` hook in order to maintain support with our overloading of `print`.

Note that there are parts of galgebra that use plaintext strings containing latex, which is why this output still contains latex.

---

First CI run will fail, using it to identify which notebooks need rebuilding.

### CODE REVIEW

File: examples/ipython/colored_christoffel_symbols.ipynb [hunk 1]
```diff
@@ -81,7 +81,7 @@
     {
      "data": {
       "text/latex": [
-       "$\\displaystyle g =\\left[\\begin{matrix}1 & 0 & 0\\\\0 & 1 & 0\\\\0 & 0 & 1\\end{matrix}\\right]$"
+       "$\\displaystyle g =\\left[\\begin{array}{ccc}1 & 0 & 0\\\\0 & 1 & 0\\\\0 & 0 & 1\\end{array}\\right]$"
```
Comment (eric-wieser) [inline examples/ipython/colored_christoffel_symbols.ipynb:84]: These changes are caused by swapping to the default sympy printer to our printer.

**Comment by eric-wieser** [comment]
sympy 1.3 CI is giving me grief, let's see if I managed to fix it. Will merge once it passes.

**Comment by eric-wieser** [comment]
Ci issues will be fixed by gh-372, will rebase after that one is merged.

**Comment by utensil** [comment]
> sympy 1.3 CI is giving me grief

When would you plan to release? Then we can drop support for 1.3. We might need to do it before May ends anyway?

**Comment by eric-wieser** [comment]
I think there are a couple of printing bugs open I'd like to close first, but otherwise I'd be keen to release soon.

I might do a final sweep for deprecations too.

**Comment by eric-wieser** [comment]
Split #373 out of this PR too, since it doesn't really fit with the rest of the PR.

**Pull request merged**